### ISSUE TITLE
Support for IRI 1.4.1.4

### ISSUE THREAD

**Issue by alberk8** [iotaledger/iota.lib.csharp#10]
Is an update coming to support the latest IRI version?  Thank you.

**Comment by VelkyTlustoprd** [comment]
I would be willing to work on that, but I have no clue as I'm new to this iota stuff. What needs to be done?

**Comment by alberk8** [comment]
I am not sure,  I am new to IOTA too. The only thing that stands out is that there is no code commit like 100 months vs the java, go, js and other languages.

**Comment by chrisoro** [comment]
Seems the cs version has no maintainer and is pretty dead communitywise

**Comment by alberk8** [comment]
Yes, there are more people in Gobi desert than here. 😀

**Comment by engst03** [comment]
Yes, what a pity. How can IOTA grow if basic libraries are not maintained?
